lib/rucio/daemons/replicarecoverer/suspicious_replica_recoverer.py

```diff
@@ -197,7 +193,7 @@ def run_once(heartbeat_handler: Any, younger_than: int, nattempts: int, vos: Opt
 
             # RSEs that aren't available shouldn't have suspicious replicas showing up. Skip to next RSE.
             if not rse['availability_read'] and ((len(suspicious_replicas_avail_elsewhere) > 0) or (len(suspicious_replicas_last_copy) > 0)):
-                logger(logging.WARNING, "%s is not available (availability_read: %s), yet it has suspicious replicas. Please investigate. \n", rse_expr, rse['availability_read'])
+                logger(logging.WARNING, "%s is not available (availability: %s), yet it has suspicious replicas. Please investigate. \n", rse_expr, rse['availability'])
```

Comment:
Why did you change this back to `availability`? This will be removed this release so we would have to change it back again then.

Reply:
I'm not sure what happened there - it still says `availability_read` on my end. I've fixed it.